azure-mgmt-resources/src/main/java/com/microsoft/azure/management/resources/fluentcore/utils/Utils.java

```diff
@@ -38,6 +38,19 @@ public static boolean toPrimitiveBoolean(Boolean value) {
     }
 
     /**
+     * Converts an object integer to a primitive int.
+     *
+     * @param value the <tt>Integer</tt> value
+     * @return 0 if the given Integer value is null else the primitive value
+     */
+    public static int toPrimitiveInt(Integer value) {
```

Comment:
I'm not sure treating null as 0 is desirable. If `i` is an `Integer` instance, why not just return `i.intValue()`, without any special `Utils` function? It seems if `Integer` is `null`, it'd throw a NPE, which one could argue is the proper thing to do anyway, no?